app/js/views/reminders.jsx

```diff
@@ -101,6 +105,70 @@ export default class Reminders extends React.Component {
       });
   }
 
+  /**
+   * Generate a phrase to be spoken to confirm a reminder.
+   *
+   * @param {Object} reminder
+   * @returns {string}
+   */
+  confirmAReminder(reminder) {
+    const users = reminder.recipients
+      .map((user) => user === 'me' ? 'you' : user)
+      .join(' and ');
```

Comment:
Could just use a very simple `reduce` for now?
 `.reduce((str, name, index) => str + ((index === x.length - 1) ? ', and ' : ', ') + name);`